source/administration/image-proxy.rst

```diff
@@ -11,6 +11,8 @@ third-party servers.
 Proxy servers also provide a layer of caching, and can be made faster and more reliable than third-party sites. This caching 
 also helps preserve posts by protecting them from dead images.
 
+When enabled, the image proxy needs to be publicly accessible to the Mattermost client, which requests the image directly from the image proxy.
```

Comment:
From a followup to that conversation, we should probably mention that it needs to be accessible by both clients and the server.

```suggestion
When enabled, the image proxy needs to be publicly accessible to both the Mattermost client and server.
```

I'm not sure how to make the second part of the sentence work if we mention both of those though